runtime/queries/nix/injections.scm

```diff
@@ -129,3 +129,35 @@
   ])
   (#match? @_func "(^|\\.)writeRust(Bin)?$")
   (#set! injection.combined))
+
+;; (nixosTest) testScript
+((binding
+  attrpath: (attrpath) @_attr_name 
+    (#eq? @_attr_name "nodes")
+  )
+  (binding
+   attrpath: (attrpath) @_func_name (#eq? @_func_name "testScript")
+   (_
+     (string_fragment) @injection.content (#set! injection.language "python")
+   ))
+   (#set! injection.combined))
+
+;; home-manager Neovim plugin config
+(attrset_expression
+  (binding_set
+    (binding
+      attrpath: (attrpath) @_ty_attr
+      (_
+        (string_fragment) @_ty
+      )
```

Comment:
```suggestion
      (_
        (string_fragment) @_ty)
```

Same for all other parentheses that I don't want to list out

Reply:
I think that should be all of them.